ee_tests/cico_run_e2e_tests.sh

```diff
@@ -70,6 +70,7 @@ docker run --shm-size=256m --detach=true --name="$CONTAINER_NAME" --cap-add=SYS_
           -e QUICKSTART_NAME -e RELEASE_STRATEGY -e RESET_ENVIRONMENT -e TEST_SUITE \
           -e ZABBIX_ENABLED -e ZABBIX_HOST -e ZABBIX_METRIC_PREFIX \
           -e ZABBIX_PORT -e ZABBIX_SERVER \
+          -e "FORCE_COLOR=1" \
```

Comment:
I would like to keep them ordered alphabetically.

Reply:
OK, I didn't realize we did :-)